File: week1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        //Prompt to enter number 1-8
        n = get_int("Height: ");
    }
    while (n <= 0 || n > 8);

    for (int i = 0; i < n; i++)
    {
        //Print spaces
        for (int j = 0; j < (n - i) - 1; j++)
        {
            printf(" ");
        }

        // Print grid of bricks
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/pset6/sentimental-mario-less/mario.py
while True:
    try:
        # Prompt to enter number 1-8
        n = int(input("Height: "))
        if n > 0 and n <= 8:
            break
    except ValueError:
        print("Invalid input. Please enter a positive integer less than or equal to 8.")

for i in range(n):
    # Print spaces
    for j in range(n - i - 1):
        print(" ", end="")
    # Print grid of bricks
    for k in range(i + 1):
        print("#", end="")
    print("\n", end="")